boards/sifli/sf32lb52_devkit_lcd/sf32lb52_devkit_lcd.yaml

```diff
@@ -12,4 +12,5 @@ toolchain:
 supported:
   - uart
   - gpio
+  - ble
```

Comment:
This is a problem:
```
❯ git grep -- '- ble' boards|wc -l
      82
❯ git grep -- '- bluetooth' boards|wc -l
     288
```

The inconsistency should be fixed and we should just standardize on one of the above. The shortest path would be to eliminate ble, so let's at least not add more of them. I.e. use bluetooth instead.